frontend/src/types/analytics.ts

```diff
@@ -37,8 +37,8 @@ export type Events = {
   CLICK_HOME_GALLERY_IMAGE: {
     /** the set to which the image belongs */
     set: string
-    /** the identifier of the image */
-    identifier: string
+    /** the id of the image */
```

Comment:
```suggestion
    /** The unique ID of the media */
```

Reply:
Thanks for encouraging me. Suggested changes have been applied.